css/ampshared.css

```diff
@@ -228,7 +229,8 @@ i-amphtml-sizer {
 }
 
 .i-amphtml-notbuilt,
-[layout]:not(.i-amphtml-element)
+[layout]:not(.i-amphtml-element),
+[width][height][sizes]:not([layout]):not(.i-amphtml-element)
```

Comment:
Add a comment that this is "implicit responsive" layout? 

Also, looks like `[sizes]` or `[heights]` will qualify: https://github.com/ampproject/amphtml/blob/67815dc7b6c1f434a09fcd52227a01f7b0b2c728/src/layout.js#L438-L440

Reply:
Oo, that's a good point. I'll update.